fastplotlib/layouts/_base.py

```diff
@@ -12,14 +12,16 @@
 from ..graphics._base import Graphic, GraphicCollection
 from ..graphics.selectors._base_selector import BaseSelector
 
+from ..utils import GraphicMethods
+
 # dict to store Graphic instances
 # this is the only place where the real references to Graphics are stored in a Python session
 # {hex id str: Graphic}
 GRAPHICS: Dict[str, Graphic] = dict()
 SELECTORS: Dict[str, BaseSelector] = dict()
 
 
-class PlotArea:
+class PlotArea(GraphicMethods):
```

Comment:
mixin to subplot, not here, we want to keep `PlotArea` as a very bare base class